.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * 1'
```

Comment:
why the cron ?
and which branch will it built ? all of them ? :-)

Reply:
Cron will only build the main branch of the repository.

I like to rebuild periodically, so issues with dependencies (that might be removed from a repository) or issues with the build environment (ubuntu-latest will soon be updated to a newer release) will be detected early.

Else, you would only notice when you try the next build, which might be a long time after the breaking change has occured, and thus making it harder to debug it.